browser/resources/settings/brave_leo_assistant_page/model_list_section.ts

```diff
@@ -77,7 +77,7 @@ class ModelListSection extends ModelListSectionBase {
     const isEditing = this.isEditingModelIndex_ !== null
 
     // Since model-config-ui is conditionally rendered, we use this.$$ API to access the element
-    const modelConfigElement = this.$$('#model-config-ui') as any
+    const modelConfigElement: any = this.$$('#model-config-ui')
```

Comment:
I suspect this might compile without the any, if its triggering a type assertion.